3rdparty/openssl/include_3/crypto/dso_conf.h

```diff
@@ -0,0 +1,18 @@
+/* WARNING: do not edit! */
+/* Generated by Makefile from openssl_3/include/crypto/dso_conf.h.in */
+/*
```

Comment:
From what I understand, the files under include_3 are identical to the ones under include for openssl 1.1.1?

Reply:
It does look like bn_conf.h and dso_conf.h are identical between include/ and include_3/.

We can't have all OpenSSL 1.0 and OpenSSL 3.0 header generated files in the same include directory, because many have the same name but different configurations. We could seperate files into include_1/, include_3/, include_shared/ for example.

However, I think this is better future proofing. For example, we could update OpenSSL version again that could add differences to these files. Or we could change configuration flags in update.make that cause a change between these files.